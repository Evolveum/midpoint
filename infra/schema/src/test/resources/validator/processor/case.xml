<case xmlns="http://midpoint.evolveum.com/xml/ns/public/common/common-3"
        oid="10a2c2b4-4d02-4f52-a4ac-092d8e321006">

    <name>case</name>
    <taskRef oid="c5baeaf3-e917-4d1a-9aed-8a4bc147f5f9"/>
    <modelContext>
        <state>execution</state>
        <projectionContext id="1">
            <accountPasswordPolicyRef>
                <object>
                    <stringPolicy>
                        <description>123</description>
                    </stringPolicy>
                </object>
            </accountPasswordPolicyRef>
        </projectionContext>
        <projectionContext id="2">
            <accountPasswordPolicyRef oid="2db718b6-243a-11e7-a9e5-bbb2545f80ed"/>
        </projectionContext>
    </modelContext>
</case>
