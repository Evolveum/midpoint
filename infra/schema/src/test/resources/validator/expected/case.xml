<case xmlns="http://midpoint.evolveum.com/xml/ns/public/common/common-3"
        oid="10a2c2b4-4d02-4f52-a4ac-092d8e321006">

    <name>case</name>

    <modelContext>
        <state>execution</state>
        <projectionContext id="1"/>
        <projectionContext id="2"/>
    </modelContext>
</case>
